pkg/autoscaler/statforwarder/processor.go

```diff
@@ -101,6 +101,7 @@ func (p *bucketProcessor) process(sm asmetrics.StatMessage) error {
 		if err != nil {
 			return err
 		}
+		p.setConn(c)
```

Comment:
probably needs a test, that verified this? :)

Reply:
done